create_derivatives/models.py

```diff
@@ -11,17 +11,24 @@ class Bag(BasePackage):
     MANIFESTS_CREATED = 4
     UPLOADED = 5
     CLEANED_UP = 6
+    TIFF_PREPARED = 7
+    PDF_COMPRESS = 8
+    PDF_OCR = 9
     PROCESS_STATUS_CHOICES = (
         (CREATED, "Created"),
         (PREPARED, "Prepared"),
         (JPG2000, "JPG2000 derivatives created"),
         (PDF, "PDF derivatives created"),
         (MANIFESTS_CREATED, "Manifests created"),
         (UPLOADED, "Derivatives and manifests uploaded to AWS"),
-        (CLEANED_UP, "Files removed from temp directory"))
+        (CLEANED_UP, "Files removed from temp directory"),
+        (TIFF_PREPARED, "TIFFs prepared for conversion"),
+        (PDF_COMPRESS, "PDF compressed"),
+        (PDF_OCR), "PDF OCRed")
```

Comment:
I think the `)` may be in the wrong place?

Reply:
🤦